teaching_reproducibility.Rmd

```diff
@@ -275,65 +334,59 @@ on *The Role of Theory in Data Analysis* by Roger Peng:
 > 
 > *Within that one statement, we can "compress" over 20 years of experience.*
```

Comment:
Do you think this quote goes a bit against our previous section on how we want them to fail controllably?

Reply:
I don't think so. I think it's more aligned. The letting them fail in a controlled manner is not letting them fail over a graduate degree.